Added initial English localization for documentation (#13)
furiosa-ai/furiosa-sdk: docs/en/source/installation/driver.rst

*************************************************
NPU Kernel Driver and Firmware Installation
*************************************************

> FuriosaAI provides the kernel driver and firmware for Alveo U250 and AWS F1 evaluation environments.

Reviewer: ```suggestion
Currently, FuriosaAI provides the kernel driver and firmware of Alveo U250 and AWS F1.
```

Mentioning 'evaluation environment' in Korean document wasn't necessary. I'll reflect this change to Korean document too.

> Select one of the two options below based on the environment being evaluated. (?*)

Reviewer: ```suggestion
Select one of the two options below based on your hardware.
```

  * :ref:`Alveo U250`
  * :ref:`AWS F1`

.. _Alveo U250:

Alveo U250 kernel driver and firmware installation
****************************************

.. code-block::

  $ git clone https://github.com/furiosa-ai/furiosa-fpga-install.git
  $ cd furiosa-fpga-install
  $ sudo ./install_furiosa_fpga_u250

  Furiosa AI's F1 SDK has been successfully installed. 
  Please REBOOT this machine to complete the installation.

> Use the ``check_fpga_device`` command (included in the ``furiosa-fpga-install`` repository) 

Reviewer: ```suggestion
Please run the ``check_fpga_device`` command (included in the ``furiosa-fpga-install`` repository) 
```

to check if the installation was successful.

.. code-block::

  $ cd furiosa-fpga-install
  $ ./check_fpga_device
  [OK] Furiosa AI's FPGA device is detected.

.. _AWS F1:

AWS F1 kernel driver and firmware installation
****************************************

Minimum Requirements
---------------------------------
* Permission to access the AWS F1 FPGA image for evaluation provided by FuriosaAI. 

>   * You can receive permission by sharing your AWS account ID with FuriosaAI.

Reviewer: It seems to need more indent. Rendered pages messed up the bullet list. This problem was originated from the original Korean document. You can see the problem if you generate rendered pages. I'll reflect the change to Korean document too.
